.vscode/cspell.json

```diff
@@ -203,7 +203,6 @@
     "sdk/tools/azure-sdk-archetype/**",
     "sdk/tools/azure-sdk-build-tool/**",
     "sdk/storage/azure-storage-file-share/**",
-    "sdk/translation/azure-ai-documenttranslator/**",
```

Comment:
We should follow the guidelines on deprecating a beta package that has no GA release.

https://eng.ms/docs/products/azure-developer-experience/develop/sdk-deprecation-guidelines

cc: @sandeep-sen